Dockerfile.dev

```diff
@@ -1,15 +1,29 @@
 FROM ubuntu:16.04
 
 RUN apt-get update
-RUN apt-get install -y git
-RUN apt-get install -y wget curl
-RUN apt-get install -y python3-pip
+RUN apt-get install -y git wget curl bzip2 build-essential
 
-RUN pip3 install --upgrade pip
-RUN pip3 install grpcio-tools
-RUN pip3 install tensorflow==2.0.0-alpha0
-RUN pip3 install mysql-connector-python
-RUN pip3 install pyhive
+# Miniconda - Python 3.6, 64-bit, x86, latest
```

Comment:
If we are duplicating some code in .Dev, can we group the duplicated ones and add some notes in case when they need to be synchronized?